packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -489,6 +501,10 @@ abstract final class SystemChrome {
   /// }
   /// ```
   ///
+  /// If your Flutter app is targeting Android 16 (API 36) or later and you are using
```

Comment:
I think under a code block will make this hard to find documentation. Why not put this with the other information about limitations on android ?
``` /// ## Limitations
  ///
  /// ### Android
```